buildSrc/src/main/groovy/com/google/android/perftesting/PerfTestTaskGeneratorPlugin.groovy

```diff
@@ -119,13 +119,17 @@ public class PerfTestTaskGeneratorPlugin implements Plugin<Project> {
                         outputLine = outputLine.trim()
                         if (!outputLine.startsWith("List of devices attached") && !"".equals(outputLine)) {
                             String[] lineParts = outputLine.split(/\s+/) // The regex groups whitespace.
-                            if (lineParts.length != 6) {
+                            if (lineParts.length > 6 || lineParts.length < 5 ) {
```

Comment:
The algorithm here is too weak. We have suffered from this issue twice, and it has to be improved.